### PR TITLE
Calling the base on_mob_life is mandatory

### LINKED ISSUE #27983: Chloral Hydrate appears to be worthless, as are sleepypens
The recent change swapped out the morphine for the chloral but now all it does is moderate stamina damage and a mute, no sleep, no stun, nothing.

### LINKED ISSUE #29354: Delayed chloral does not metabolize
It doesn't appear that delayed chloral (as used in sleepypen) metabolizes correctly.

### PR DESCRIPTION
**Pull request by Supermichael777**
:cl:
fix: delayed chloral hydrate actually works now.
/:cl:

fixes #29354
fixes #27983

on_mob_life should never return always use ..() beforhand or set it up so that its an if-else tree instead of a series of ifchecks with hard returns.

so the thing that ticks current cycle is the base of on_mob_life. additionally fixes a second coding error where this would have also acted like normal CH because inheritance. Chems with custom life effects like this can't be inherited from by chems with a different one.

thankfully the chem id system means that the path of chems mean nothing.

TLDR: don't put in returns and don't have parents for IRL logic instead of code reasons.

### CODE REVIEW

**Comment by Supermichael777** [comment]
@kevinz000 150 hours of testing: https://github.com/tgstation/tgstation/blob/67456fb6a19b88a1cccd52c7f7a8b7d72c2ad7c8/code/modules/reagents/chemistry/reagents/toxin_reagents.dm

**Comment by kevinz000** [comment]
woops sorry.

**Comment by Supermichael777** [comment]
note its also goofs fault

**Review by Cyberboss — APPROVED** [review]
